File: DIP/Spoilers/MarsRoverKata/Behaviors/DoNothing.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MarsRoverKata.Behaviors
{
    public class DoNothing : IBehavior
    {
        public Alien Parent { get; set; }
        public void ExecuteBehavior()
        {
            
        }
    }
}

File: DIP/Java_Final/MarsRoverKata/src/main/java/MarsRoverKata/Behaviors/DoNothing.java
package MarsRoverKata.Behaviors;

import MarsRoverKata.Alien;

public class DoNothing implements IBehavior {
    private Alien parent;

    public Alien getParent() { return parent; }

    public void setParent(Alien parent) { this.parent = parent; }

    public void executeBehavior() {

    }
}
